Verlust von Süßem

Pegasus01123 opened the pull request:
Zweimal Verlust eingefügt bei Skip.

Pegasus01123 committed:
Verlust von Süßem

Zweimal Verlust eingefügt bei Skip.

dnns01 commented on data/story.json:
```diff
@@ -50,7 +50,19 @@
-      }
+      },
+      {
+        "text": "Während ihr weitergeht hört ihr es in euren Süßigkeiten rascheln. Als ihr nachschaut schaut euch eine Spinne aus großen Augen an. \nVor Schreck lasst ihr die Süßigkeiten fallen.",
```
"... aus großen Augen an"? 
Und ich denke eher "Vor Schreck lasst ihr einen Teil eurer Süßigkeiten fallen"

dnns01 commented on data/story.json:
```diff
@@ -50,7 +50,19 @@
+        "sweets_max": -5
+      },
+      {
+        "text": "Beim weitergehen stellt ihr fest, dass euer Süßigkeiten Behälter ein Loch hat. Als ihr zurückschaut stellt ihr fets, dass einige Kinder hinter euch herlaufen und eure Süßigkeiten einsammeln.",
```
"Beim Weitergehen"
"stellt ihr fest"

Pegasus01123 committed:
Update story.json

dnns01 commented on data/story.json:
```diff
@@ -52,13 +52,13 @@
       },
       {
-        "text": "Während ihr weitergeht hört ihr es in euren Süßigkeiten rascheln. Als ihr nachschaut schaut euch eine Spinne aus großen Augen an. \nVor Schreck lasst ihr die Süßigkeiten fallen.",
+        "text": "Während ihr weitergeht hört ihr es in euren Süßigkeiten rascheln. Als ihr nachschaut seht ihr eine Große Spinne über eure Süßigkeiten krabbeln. \nVor Bei dem Versuch die Spinne loszuwerden verliert ihr einen Teil eurer Süßigkeiten.",
```
"große Spinne"
"Vor Bei dem Versuch"?

Pegasus01123 committed:
requested Changes, kleinere Änderungen Rechtschreibfehler und eine weitere Tür.

Pegasus01123 committed:
Eine neue Tür, Rechtschreibung

Pegasus01123 committed:
Merge remote-tracking branch 'origin/3-süßigkeiten-verlieren-spinne-krabbelt-in-tüte-und-man-kippt-was-aus-oder-so-=-story' into 3-süßigkeiten-verlieren-spinne-krabbelt-in-tüte-und-man-kippt-was-aus-oder-so-=-story

dnns01 committed:
Merge branch 'develop' into 3-süßigkeiten-verlieren-spinne-krabbelt-in-tüte-und-man-kippt-was-aus-oder-so-=-story
